doc/ceph-volume/inventory.rst

```diff
@@ -0,0 +1,16 @@
+.. _ceph-volume-lvm-inventory:
+
+``inventory``
+=============
+The ``inventory`` subcommand queries a hosts disc inventory and provides
+hardware information and metadata on every physical device.
+
+By default the command returns a short, human-readable report of all physical disks.
```

Comment:
This could all be one sentence, since they are talking about the report from devices that is human-readable:

```suggestion
By default the command returns a short, human-readable report of all physical disks, but can accept a device path for more extensive information.
```